packages/venia-concept/src/components/Navigation/navigation.js

```diff
@@ -33,13 +33,21 @@ class Navigation extends PureComponent {
             userMore: string,
             userName: string
         }),
+        closeDrawer: func.isRequired,
+        completePasswordReset: func.isRequired,
+        createAccount: func.isRequired,
```

Comment:
I added `closeDrawer` and `createAccount` as required functions in `propTypes` because they are passed in and called directly in this file.